packages/mariadb/build.sh

```diff
@@ -95,6 +97,7 @@ termux_step_pre_configure() {
 		# Avoid undefined reference to __atomic_load_8:
 		CFLAGS+=" -latomic"
 	fi
+	sed -i 's/^\s*END[(][)]/ENDIF()/g' $TERMUX_PKG_SRCDIR/libmariadb/cmake/ConnectorName.cmake
```

Comment:
Why is this sed'ed twice?

Reply:
Once for host build and once for each arch build

Reply:
Oh, right, I thought both occurrences were in termux_step_host_build